boilerplates/python-lib/src/mylib/__init__.py

```
@@ -0,0 +1,7 @@
+"""mylib"""
+
+__version__ = "0.1.0"
+
+from . import main
+
+__all__ = ["main", "__version__"]
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)